web-app/pom.xml

```diff
@@ -21,8 +21,25 @@
         <maven.compiler.target>1.8</maven.compiler.target>
         <maven.test.skip>true</maven.test.skip>
     </properties>
+    <repositories>
```

Comment:
This should be unnecessary as it is one of the default repo in Maven.

Reply:
This is documented in our official document https://docs.microsoft.com/en-us/azure/application-insights/app-insights-java-get-started. I prefer to keep them to make sure our demo align with our doc to not confuse our customers. And I am open to remove it too. Let me know your suggestions.

Reply:
Thanks for the contribution. Like to know if this is an optional or required step?

Reply:
It is optional. If you don't specify a valid instrumentation key, AI will be disabled by default.